src/CoreWCF.Primitives/src/CoreWCF/IdentityModel/Selectors/EmptySecurityTokenResolver.cs

```diff
@@ -0,0 +1,15 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using CoreWCF.IdentityModel.Tokens;
+using CoreWCF.Security;
+
+namespace CoreWCF.IdentityModel.Selectors
+{
+    static class EmptySecurityTokenResolver
+    {
+        static readonly SecurityTokenResolver _instance 
+            = SecurityTokenResolver.CreateDefaultSecurityTokenResolver( EmptyReadOnlyCollection<SecurityToken>.Instance, false );
```

Comment:
Although it's going away, you missed the accessibility modifier on this field. Also the name should start with s_